src/main/java/brooklyn/entity/proxy/aws/ElbControllerImpl.java

```diff
@@ -173,175 +174,179 @@ protected void startLoadBalancer() {
-            ELBApi elbApi = api(loadBalancerServiceContext);
-            String elbDnsName = elbApi.getLoadBalancerApi()
-                    .createListeningInAvailabilityZones(
-                            elbName,
-                            Listener.builder()
-                                    .protocol(Protocol.HTTP)
-                                    .instancePort(instancePort)
-                                    .port(loadBalancerPort)
-                                    .build(),
-                            availabilityZoneNames);
+            String elbDnsName = "";
+            Listener listener = buildListener();
+//          create vanilla load balancer
+            if(availabilityZoneNames != null && availabilityZoneNames.size() > 0 ){
+                elbDnsName = getELBApi().getLoadBalancerApi().createLoadBalancerInAvailabilityZones(elbName, ImmutableSet.<Listener>of(listener), availabilityZoneNames);
+            }
+            else if(subnets != null && !subnets.isEmpty()) {
+                elbDnsName = getELBApi().getLoadBalancerApi().createLoadBalancerInSubnets(elbName, ImmutableSet.<Listener>of(listener), subnets);
+            }
+
+            LoadBalancer loadbalancer = getELBApi().getLoadBalancerApi().get(elbName);
+            configureListeners(elbName, loadbalancer);
+            configureHealthCheck();
+            configureListeners(elbName, loadbalancer);
 
             sensors().set(Attributes.HOSTNAME, elbDnsName);
         } finally {
             Closeables2.closeQuietly(loadBalancerServiceContext.unwrap());
         }
     }
 
+    private void configureHealthCheck() {
+        String elbName = getConfig(LOAD_BALANCER_NAME);
+        Integer healthCheckInterval = getConfig(HEALTH_CHECK_INTERVAL);
+        Integer healthCheckTimeout = getConfig(HEALTH_CHECK_TIMEOUT);
+        Integer healthCheckHealthyThreshold = getConfig(HEALTH_CHECK_HEALTHY_THRESHOLD);
+        Integer healthCheckUnhealthyThreshold = getConfig(HEALTH_CHECK_UNHEALTHY_THRESHOLD);
+        String healthCheckTarget = getConfig(HEALTH_CHECK_TARGET);
+
+        final LoadBalancer loadBalancer = getELBApi().getLoadBalancerApi().get(elbName);
+        final HealthCheck healthCheck = loadBalancer.getHealthCheck();
+
+        final HealthCheck healthCheckConfig = HealthCheck.builder()
+                .healthyThreshold(healthCheckHealthyThreshold != null ? healthCheckHealthyThreshold : healthCheck.getHealthyThreshold())
+                .unhealthyThreshold(healthCheckUnhealthyThreshold != null ? healthCheckUnhealthyThreshold : healthCheck.getUnhealthyThreshold())
+                .interval(healthCheckInterval != null ? healthCheckInterval : healthCheck.getInterval())
+                .target(healthCheckTarget != null ? healthCheckTarget : healthCheck.getTarget())
+                .timeout(healthCheckTimeout != null ? healthCheckTimeout : healthCheck.getTimeout())
+                .build();
```

Comment:
the reason i did it this way was so if you used the `BIND_TO_EXISTING` option you could reconfigure the health check rather than it just using the configured health check.